Dockerfile

```diff
@@ -0,0 +1,34 @@
+# FOR ARM64, 
+FROM --platform=linux/amd64 python:3.9-slim-buster
```

Comment:
Are you aware that this is Debian 10 (current is Debian 12)?

Reply:
I've updated this to using 3.9-slim-bookworm, it uses debian 12 and is actively maintained.